query-connector/src/app/query-building.ts

```diff
@@ -0,0 +1,72 @@
+import { UUID, randomUUID } from "crypto";
+import { Concept, ValueSet } from "./constants";
+
+// TODO: Potentially merge this / infer this from the type created via the
+// database creation workstream
+export type UserQueryInput = {
+  queryName: string;
+  author: string;
+  valueSets: ValueSet[];
+  concepts?: Concept[];
```

Comment:
Sorry one more question: why do we need `concepts` here if concepts are included in the `ValueSet` type?

Reply:
I included this to allow for additional insertions to the `query_included_concepts` table in the future where we might have concepts without parent valuesets. Probably won't use this for a while until users can add their own codes, but just included it for completeness since we have that table already

Reply:
Missed this comment earlier. We shouldn't have concepts without valuesets. If a user wants to create a concept, e.g., add a single code, they would still need to make a valueset for that single concept.

Reply:
Ahh ok! Will go back and remove this then. Glad we caught it